lib/toolbox-overlay.js

```diff
@@ -42,6 +44,15 @@ const ToolboxOverlay = Class(
 
     Trace.sysout("ToolboxOverlay.initialize;", options);
 
+    // handler of packetCache toggling
+    this.onToggleCache = () => {
+      this.packetCache = prefs.packetCacheEnabled ? [] : null;
+    };
```

Comment:
Please define the method the same way as the other methods (like e.g. onSendPacket)

Honza